src/components/header.js

```diff
@@ -1,16 +1,18 @@
 /** @jsx jsx */
 import { Link } from 'gatsby';
-import { jsx } from 'theme-ui';
+import { jsx, useColorMode } from 'theme-ui';
 import { useSiteMetadata } from '../hooks/use-site-metadata';
 import { Button } from './button';
 import { DesktopOnly } from './desktop-only';
 import { NavLink } from './nav-link';
 import { Icon } from './icon';
 // eslint-disable-next-line
 import React from 'react';
+import { FaMoon, FaSun } from 'react-icons/fa';
```

Comment:
`react-feather` already have soon and moon icons. https://bit.dev/feathericons/react-feather

Any reason you use another library?

Reply:
@malcolm-kee , I didn't realize `react-feather` have it.

Then it's even better. I'll update it to use the existing `react-feather` library.